pages/4_Documentation.py

```diff
@@ -323,6 +323,35 @@
          ( 'N.A.'           , 'The mutation site could not be predicted (for instance, because it is located in an unstructured region)') ]
 st.table(pd.DataFrame(data, columns=['Value', 'Meaning']))
 
+st.subheader("""Long range: AlloSigma2-PSN""")
+
+st.write("""This module is available only for ensemble mode and is designed to 
+validate the predictions of AlloSigma2 using a Protein Structure Network (PSN)-based method.
+
+The module relies on the output of the Long Range module, specifically the mutations predicted 
+to affect pocket residues. These predictions are then used as input for identifying potential 
+communication paths between mutations and their respective pocket sites using PyInteraph2. 
+We retain only paths that are 4 residues in length or more. If such a path is identified, 
+it is considered a validation of the AlloSigma2 predicted allosteric effect of a mutation
+on the pocket residue(s).
+
+The module aims to leverage these two methods to validate the predictions and use a 
+consensus approach to build the final classification of a mutation's long-range effects.
+
+The dataset columns generated by this module are:""")
+
+data = [ ( "AlloSigma2-PSN classification",
+           "Classification of long range AlloSigma2-PSN effects",
+           "see below") ]
+st.table(pd.DataFrame(data, columns=['Column', 'Description', 'Possible values']))
+
+st.write("""The classification column can have the following values:""")
+
+data = [ ( 'damaging'       , 'The mutation effect on pocket site(s) was identified by both methods'),
+         ( 'neutral'        , 'The mutation was not predicted to have any effect on pocket site(s)'),
+         ( 'uncertain'      , 'The the two methods are not in agreement, or the mutation could not be assessed by AlloSigma2') ]
```

Comment:
would be better to explain a bit more in depth what "could not be assessed by" - the way it's written here sounds like a technical error of some sorts, while it is a conscious decision to not to classify residues whose mutation has a too small change in volume